### PR TITLE
Submit to Treeherder; bump shared-library version

### PR DESCRIPTION
**Pull request by stephendonner**
@davehunt / @m8ttyB / @oremj r?

Ad-hoc run here: https://fx-test-jenkins-dev.stage.mozaws.net:8443/job/bouncer.adhoc/18/console

For the full log: https://gist.github.com/stephendonner/1c135b945f2d380b0e9c277c26b078bd

Background blog post: http://davehunt.co.uk/2017/04/10/reporting-test-results-to-treeherder.html

### CODE REVIEW

File: tests/e2e/tox.ini [hunk 1]
```diff
@@ -7,7 +7,8 @@ passenv = PYTEST_ADDOPTS PYTEST_BASE_URL JENKINS_URL JOB_NAME BUILD_NUMBER
 deps = -rrequirements/tests.txt
 commands = pytest \
   --junit-xml=results/{envname}.xml \
-  --log-raw=results/{envname}.log \
+  --log-raw=results/{envname}.txt \
```
Comment (davehunt) [inline tests/e2e/tox.ini]: This should be named `results/{envname}_raw.txt`

File: tests/e2e/Jenkinsfile [hunk 2]
```diff
@@ -24,7 +25,8 @@ pipeline {
         always {
           archiveArtifacts 'tests/e2e/results/*'
           junit 'tests/e2e/results/*.xml'
-          submitToActiveData('tests/e2e/results/py27.log')
+          submitToActiveData('tests/e2e/results/py27_raw.txt')
+          submitToTreeherder('go-bouncer', 'T', 'Tests', 'tests/e2e/results/py27_tbpl.txt')
```
Comment (davehunt) [inline tests/e2e/Jenkinsfile]: The arguments are project (repo), job symbol, job name, artifact path, log path. You're missing the artifact path, which should probably be 'tests/e2e/results/*'

File: tests/e2e/tox.ini (hunk 1 above)
Comment (stephendonner) [inline tests/e2e/tox.ini]: Fixed

File: tests/e2e/Jenkinsfile (hunk 2 above)
Comment (stephendonner) [inline tests/e2e/Jenkinsfile]: Thanks; fixed.

**Comment by davehunt** [comment]
I notice that https://github.com/mozilla-services/go-bouncer/issues/105 has not been resolved, so results will not be in Treeherder until that is done.

**Pull request merged**